src/RoslynAnalyzers/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -658,27 +662,29 @@ await checkHelpLinkAsync(helpLinkUri).ConfigureAwait(false))
 
                 return;
 
-                async Task<bool> checkHelpLinkAsync(string helpLink)
+                async Task<(HttpStatusCode, string? responseContent)> checkHelpLinkAsync(string helpLink)
                 {
                     try
                     {
                         if (!Uri.TryCreate(helpLink, UriKind.Absolute, out var uri))
                         {
-                            return false;
+                            // TODO: why would we not fail this if the URI is invalid?
+                            return (HttpStatusCode.OK, null);
```

Comment:
I feel like this should definitely fail (it did before as well) and not return ok